frappe/public/build.json

```diff
@@ -306,6 +306,9 @@
 		"public/js/frappe/list/list_view.js",
 		"public/js/frappe/list/list_factory.js",
 
+		"public/js/frappe/ui/filters/filters.js",
```

Comment:
filters is already there

Reply:
filter.js is there, but not filters.js
Without it, Medical Record will only show _Loading_

Reply:
it can't be part of default build. you can load it via `frappe.require`.

The perfect reason why not to write custom UI

cc @surajshetty3416 @netchampfaris